tools/gen_usb_descriptor.py

```diff
@@ -22,6 +22,8 @@
 # Digitizer works on Linux but conflicts with mouse, so omit it.
 DEFAULT_HID_DEVICES='KEYBOARD,MOUSE,CONSUMER,GAMEPAD'
 
+DEFAULT_WEBUSB_URL = 'www.circuitpython.org' # In the future, this may become a specific landing page
```

Comment:
Same comment as above re `www.` and `https://` prefix

Reply:
Same answer. OK to drop the www and the scheme is hardcoded to https:// via the bScheme field

Reply:
URL changed to omit www. in gen_usb_descriptor.py

Reply:
OK, go ahead and change this.